### ISSUE TITLE
CommandBarFlyout - Calling Show after Hide quickly doesn't actually show the floatie.

### ISSUE THREAD

**Issue by alanlai-ms** [microsoft/microsoft-ui-xaml#548]
**Describe the bug**
Calling Show after Hide quickly doesn't actually show the Floatie.

In OneNote, when we double click on a text, we show the transient floatie.
But one thing users do is double click > right click

In code, what it would do is a quick Hide then Show.

Expected = the full context menu shows
Actual = full context menu does not show.

**Steps to reproduce the bug**
Min repro:

myComandBarFlyout.ShowAt(senderElement, new FlyoutShowOptions { Position = e.GetPosition(senderElement)});
myComandBarFlyout.Hide();
myComandBarFlyout.ShowAt(senderElement, new FlyoutShowOptions { Position = e.GetPosition(senderElement)});

**Expected behavior**
The 2nd ShowAt does not show.

BUT, we found a work around... if the position in 2nd ShowAt changes by 1 pixel then the ShowAt would work.

**Version Info**
19H1, build using WUX RS5 SDK.
will update to MUX soon...

**Comment by jevansaks** [comment]
@llongley can you help take a look?

**Comment by chingucoding** [comment]
I am afraid, without WinUI 3 there isn't much we can do here. The CommandBarFlyout has the following event cycle with the repro code of this issue (note that opened doesn't even get raised):

1. Opening
1. Closing
1. Reopen
1. Closed

Even if we process the `ShowAt` in the CommandBarFlyout, the internal event cycle of Flyout seem to interrupt here and force to close that element. Even if we try to show it again, it still is closed after we called ShowAt for the second time.

Maybe I am missing something here, but I don't think there is something we could do here right now. I've created a branch on my fork with an interaction test for this issue [here](https://github.com/chingucoding/microsoft-ui-xaml/tree/user/chingucoding/commandbarflyout-showhideshow-isopen) if someone wants to work on this.
